substrait_consumer/tests/adhoc/test_adhoc_expression.py

```diff
@@ -31,13 +31,13 @@ class TestAdhocExpression:
     """
 
     @staticmethod
-    @pytest.fixture(scope="class", autouse=True)
+    @pytest.fixture(autouse=True)
     def setup_teardown_class(request):
```

Comment:
Since we're changing the scope to now just be function, let's change the fixture name to be `setup_teardown_function`

Reply:
Since there's a lot of overlap in these fixtures, I also wouldn't be opposed to moving it to conftest and having all the function tests reuse the single one.  There would be a little bit a bloat in it though since there are some initializations specific to only some test functions.  This can be handled in a different PR too if you want to do that.

Reply:
Sounds good, I agree with both points: I'll change the function name tomorrow and approve the plan to move the fixture (but will not do it in this PR).

Reply:
Done with latest commit.